clang-tools-extra/test/clang-doc/json/concept.cpp

```diff
@@ -0,0 +1,37 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: clang-doc --extra-arg -std=c++20 --output=%t --format=json --executor=standalone %s
```

Comment:
FileCheck is omitted here because since nothing is mapped, no file is created.

Reply:
Let’s try to mark it fail with filecheck. If that’s doesn’t work, I’m ok with this.